recipes/full_finetune_distributed.py

```diff
@@ -497,11 +497,10 @@ def _setup_model(
         # Ensure no params and buffers are on meta device
         training.validate_no_params_on_meta_device(model)
 
-        utils.log_rank_zero(
-            log,
-            f"Instantiating model and loading checkpoint took {time.perf_counter() - init_start:.2f} secs",
-        )
         if self._is_rank_zero:
+            log.info(
```

Comment:
Wait why can this not be extracted out?

Reply:
`init_start` is only defined for rank zero, and the string formatting happens before we check for rank zero in `log_rank_zero`